src/_cffi_src/openssl/ssl.py

```diff
@@ -755,4 +762,13 @@
 #else
 static const long Cryptography_HAS_TLSv1_3 = 1;
 #endif
+
+#if CRYPTOGRAPHY_OPENSSL_LESS_THAN_110 && !CRYPTOGRAPHY_IS_LIBRESSL
+static const long Cryptography_HAS_TLS_METHOD = 0;
+const SSL_METHOD* (*TLS_method)(void) = NULL;
```

Comment:
Why isn't this just `#define TLS_method SSLv23_method` and then TLS_method is available in all cases?

Reply:
Lack of attention on my part, I failed to pull this in from the prior PR.  Excellent catch and suggestion, I've committed and pushed.  Thank you, very much.